File: 02022023/11_74_Calling_In_Sick/python/calling_in_sick.py
from random import choice, randint

actually_sick = choice([True, False])
kinda_sick = choice([True, False])
hate_your_job = choice([True, False])
sick_days = randint(0, 10)

calling_in_sick: bool = None

if (actually_sick and sick_days > 0) or (kinda_sick and hate_your_job and sick_days > 0):
    calling_in_sick = True
    print("Today I will be calling in sick.")
else:
    calling_in_sick = False
    print("Today I shall work!")

File: 02022023/11_74_Calling_In_Sick/java/CallingInSick.java
import java.util.Random;

public class CallingInSick {
    public static void main(String[] args) {
        Random randomizer = new Random();

        boolean actuallySick = randomizer.nextBoolean();
        boolean kindaSick = randomizer.nextBoolean();
        boolean hateYourJob = randomizer.nextBoolean();
        int sickDays = randomizer.nextInt(10) + 1;

        if ((actuallySick && sickDays > 0) || (kindaSick && hateYourJob && sickDays > 0)) {
            boolean callingInSick = true;
            System.out.println("It is " + callingInSick + ", I will be calling in sick today!");
        } else {
            boolean callingInSick = false;
            System.out.println("It is " + callingInSick + ", I will not be calling in sick today!");
        };
    }
}